### ISSUE TITLE
[Functional-Social Embedded- Edit Profile]: After removing the photo "Profile photo" is still displaying.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#28]
Priority: 

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Fill all the input fields with valid inputs.
10. Navigate to ""Done"" button and press ENTER.
11. Navigate to ""Open Navigation Drawer"" button and press ENTER  (or double tap).
12. Navigate to ""My Profile"" and press ENTER  (or double tap) to open.
13. Navigate to  ""Edit Profile"" and press ENTER  (or double tap) to open.
14. Navigate to  ""Upload Photo"" and press ENTER  (or double tap) to open, Select any one option form the two options that is ""Take new photo"" and ""Choose existing photo"".
15. Navigate to ""Edit"" button and press ENTER  (or double tap) to open, Select ""Remove Photo""  and press ENTER  (or double tap) to remove photo.
16. Verify that photo is still displaying.

Expected
When user upload photo, and photo has been uploaded. Again user edit the profile and remove the photo then photo should be remove. It should not be display.

Actual
When user upload photo, and photo has been uploaded. Again user edit the profile and remove the photo, but photo is still displaying.

MAS Rule: Functional

Impact
If user remove the photo but it is still display in the profile image then user will face difficulty and will not able to understand that photo has been removed or not.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
